src/GregClient/PackageManagerClient.cs

```diff
@@ -1,148 +0,0 @@
-        {
-            m.AddHeader("Content-Type", "application/x-www-form-urlencoded");
-            m.AddHeader("X-AFC", AFC_CODE);
-        }
-
-        /// <summary>
-        /// Execute the REST request
-        /// </summary>
-        /// <param name="m"></param>
-        /// <returns></returns>
-        public Response Execute(Request m)
-        {
-            return new Response(ExecuteInternal((PackageManagerRequest)m));
-        }
-
-        /// <summary>
-        /// Execute and deserialize the request
-        /// </summary>
-        /// <param name="m"></param>
-        /// <returns></returns>
-        public ResponseBody ExecuteAndDeserialize(Request m)
-        {
-            return Execute(m).Deserialize();
-        }
-
-        /// <summary>
-        /// Execute the request and deserialize the content as dynamic type.
-        /// </summary>
-        /// <typeparam name="T">The Type of content</typeparam>
-        /// <param name="m">The request.</param>
-        /// <returns>A <see cref="ResponseWithContent{dynamic}"/> or null if there was an error
-        /// in executing the message.</returns>
-        public ResponseWithContentBody<dynamic> ExecuteAndDeserializeWithContent<dynamic>(Request m)
-        {
-            var response = this.Execute(m);
-
-            return new ResponseWithContentBody<dynamic>()
-            {
-                message = response.InternalRestReponse.StatusDescription,
-                content = JsonConvert.DeserializeObject<dynamic>(response.InternalRestReponse.Content),
-                success = true
-            };
-        }
-
-        /// <summary>
-        /// Get the file stream from response and save locally.
-        /// </summary>
-        /// <param name="gregResponse"></param>
-        /// <returns></returns>
-        public string GetFileFromResponse(Response gregResponse)
-        {
-            var response = gregResponse.InternalRestReponse;
-
-            if (!(response.ResponseUri != null && response.ResponseUri.AbsolutePath != null)) return "";
-
-            var tempOutput = System.IO.Path.Combine(FileUtilities.GetTempFolder(), System.IO.Path.GetFileName(response.ResponseUri.AbsolutePath));
-            using (var f = new FileStream(tempOutput, FileMode.Create))
-            {
-                f.Write(response.RawBytes, 0, (int)response.ContentLength);
-            }
-
-            //var md5HeaderResp = response.Headers.FirstOrDefault(x => x.Name == "ETag");
-            //if (md5HeaderResp == null) throw new Exception("Could not check integrity of package download!");
-
-            //var md5HeaderComputed =
-            //    String.Join("", FileUtilities.GetMD5Checksum(tempOutput).Select(x => x.ToString("X"))).ToLower();
-
-            //if (md5HeaderResp.Value.ToString() == md5HeaderComputed)
-            //    throw new Exception("Could not validate package integrity!  Please try again!");
-
-            return tempOutput;
-
-        }
-    }
-}
```

Comment:
??

Reply:
This class is not relevant, it was a non functioning client for a version of the package manager that never materialized written by contractors ~6 years ago?